.github/workflows/nightly_tests.yaml

```diff
@@ -138,6 +139,33 @@ jobs:
       if: always()
       run: python xpk.py cluster delete --cluster $PATHWAYS_TPU_CLUSTER_NAME --zone=us-central2-b
 
+  rc-cluster:
+    runs-on: [ubuntu-22.04]
+    concurrency: # We support one build test to run at a time currently.
+      group: nightly-rc-test-cluster-group
+      cancel-in-progress: false
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v5
+      with:
+        python-version: '3.10'
+    - uses: 'google-github-actions/auth@v2'
+      with:
+        credentials_json: '${{ secrets.GCP_SA_KEY }}'
+    - uses: google-github-actions/setup-gcloud@v2
+      with:
+        version: '>= 363.0.0'
+        install_components: 'beta,gke-gcloud-auth-plugin'
+    - name: Set Google Cloud CLI properties to a unused zone to verify --zone arg is passed properly in commands.
+      run: |
+        gcloud config set compute/zone us-east4-a
+        gcloud config get compute/zone
+    - name: Create a RayCluster-enabled XPK Cluster with 2 x v4-8 nodepools
```

Comment:
I saw a fix to the nightly tests just submitted: https://github.com/AI-Hypercomputer/xpk/pull/225/files, can you also add the "--network=${{secrets.NETWORK_NAME}}" flag here to contain that fix in your tests as well

Reply:
Done